src/subplots.ts

```diff
@@ -2045,3 +2075,78 @@ export class Histogram extends Frame {
     return [mouse3X, mouse3Y];
   }
 }
+
+
+export class newScatter extends Frame {
+  public points: newPoint2D[] = [];
+  public tooltipAttr: string[];
```

Comment:
Consider fully naming variables. This could be tooltipAttraction or tooltipAttrition. (i know, this is a bit far-fetched, but still, I think it is important to properly name variables)

Reply:
ok